static/src/javascripts/projects/common/views/ui/notifications-follow-link.html

```diff
@@ -1,10 +1,10 @@
- <button class="button button--large button--tertiary js-notifications__button notifications__button notifications-follow-input--solo notifications__button--<%= isSubscribed ? 'subscribed' : 'unsubscribed' %>" data-link-name="live-blog-notifications-turned-<%= isSubscribed ? 'off' : 'on' %>">
-     <%= icon %>
+ <button class="js-notifications__toggle notifications__toggle notifications-follow-input--solo notifications__toggle--<%= isSubscribed ? 'subscribed' : 'unsubscribed' %>" data-link-name="live-blog-notifications-turned-<%= isSubscribed ? 'off' : 'on' %>">
      <% if (isSubscribed) { %>
-     <span class="live-notifications__label live-notifications__label--subscribed">Alerts on for this story</span>
+         <div class="live-notifications__slider live-notifications__slider--subscribed"></div>
+         <span class="live-notifications__label live-notifications__label--subscribed">Receive updates on this story</span>
      <% } else { %>
-     <span class="live-notifications__label live-notifications__label--unsubscribed">Get alerts for this story</span>
+         <div class="live-notifications__slider live-notifications__slider--unsubscribed"></div>
+         <span class="live-notifications__label live-notifications__label--unsubscribed">Receive updates on this story</span>
      <% } %>
  </button>
-
-
+ <span class="live-notifications__label live-notifications__label--denied">Oops! You need to <a href="https://support.google.com/chrome/answer/3220216">unblock notifications</a> for www.theguardian.com</span>
```

Comment:
We are just testing with Chrome at the moment as it has the best support on mobile devices to receive notifications. The backend has been setup to support sign-ups from other browsers so if we think we need to expand to more browsers we just need to remove the restriction and ensure it works correctly.